docs/installation.md

```diff
@@ -0,0 +1,43 @@
+# Getting setup with execution environment support on:
```

Comment:
Once https://github.com/ansible/ansible-navigator/pull/536 is in, add an `installation` entry into `docs/index.md`, under one of the `toctree` directives.